uwblueprint/planet-read
# Remove custom styling of outline variant

## Notion ticket link



https://www.notion.so/uwblueprintexecs/a658c933d18c48d4b43fd99bda2bd3b4?v=ba22ac93d09040ff8219c60ac7caabdb&p=390ca5b7ae644710999acaf3563758ef

`outline` was styled specifically to blue button specs. move that styling to a separate variant so default chakra ui `outline` works as expected


## Implementation description

- swapped `outline` for `blueOutline`
- added `outline` to red delete buttons in managestorytranslation and userprofile


## Steps to test

1. open webapp
2. look at Login page, StoryCard, ManageStoryTranslationPage, UserProfilePage, and WIPComment and observe that all the secondary blue buttons look correct
3. look at the red buttons on ManageStoryTranslationPage and UserProfilePage 



## What should reviewers focus on?

- does everything look lit

## Checklist
